src/samplerate.rs

```diff
@@ -95,29 +95,58 @@ impl Samplerate {
         }
     }
 
-    /// Perform a samplerate conversion of the given input data.
-    /// If the number of channels used was not `1` (Mono), the samples are expected to be stored
-    /// interleaved.
-    pub fn process(&self, input: &[f32]) -> Result<Vec<f32>, Error> {
+    fn _process(&self, input: &[f32], output_len: usize, end_of_input: bool) -> Result<Vec<f32>, Error> {
```

Comment:
Please find a more meaningful name.

Reply:
This is just an internal function. I've tried to find how these should be named in Rust but I couldn't find anything so I went with the good ol' underscore. What do you think would be more appropriate? Something like `process_internal`?